src/components/sdk/VideoTileGrid/VideoTileGrid.stories.mdx

```diff
@@ -5,7 +5,7 @@ import { VideoTileGrid } from './';
 
 # VideoTileGrid
 
-The `VideoTileGrid` component renders all meeting session video tiles in a responsive grid layout. This includes the local tile, remote tiles, and content share tile.
+The `VideoTileGrid` component renders all meeting session video tiles in a responsive grid layout. This includes the local tile, remote tiles, and content share tile. By default, a user joins without video, so in order to see a video tile appear, you must first turn on a video tile.
```

Comment:
I think builders were expecting the video grid to appear. Maybe we could add something like: in order for the VideoTileGrid to render, at least one video tile must be shared.

Reply:
cool, updated